subsys/bluetooth/audio/shell/bap.c

```diff
@@ -3970,7 +4003,7 @@ static int cmd_print_ase_info(const struct shell *sh, size_t argc, char *argv[])
 	"[bcast_flag]" HELP_SEP "[extended <meta>]" HELP_SEP "[vendor <meta>]]"
 
 SHELL_STATIC_SUBCMD_SET_CREATE(
-	bap_cmds, SHELL_CMD_ARG(init, NULL, NULL, cmd_init, 1, 0),
+	bap_cmds, SHELL_CMD_ARG(init, NULL, NULL, cmd_init, 1, 2),
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/connectivity/bluetooth/api/audio/shell/bap.rst needs to be updated to reflect this change